pkg/commands/secretstore/root_store.go

```diff
@@ -0,0 +1,31 @@
+package secretstore
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/cmd"
+	"github.com/fastly/cli/pkg/config"
+)
+
+// RootNameStore is the base command name for secret store operations.
+const RootNameStore = "secret-store"
+
+// NewStoreRootCommand returns a new command registered in the parent.
+func NewStoreRootCommand(parent cmd.Registerer, globals *config.Data) *StoreRootCommand {
+	var c StoreRootCommand
+	c.Globals = globals
+	c.CmdClause = parent.Command(RootNameStore, "Manipulate Fastly secret stores")
+	return &c
+}
+
+// StoreRootCommand is the parent command for all 'store' subcommands.
+// It should be installed under the primary root command.
+type StoreRootCommand struct {
+	cmd.Base
+	// no flags
+}
+
+// Exec implements the command interface.
+func (c *StoreRootCommand) Exec(_ io.Reader, _ io.Writer) error {
+	panic("unreachable")
+}
```

Comment:
I think the comments should be generic to match the rest of the code base ([example](https://github.com/fastly/cli/blob/main/pkg/commands/acl/root.go)).

Reply:
I've updated the comments to match other commands. The "root" commands in particular cannot exactly match others, since there are 2 secret store "root" commands and the generic comment would not make sense because of that.